src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
+Theia Blueprint assembles a selected subset of existing Theia features and extensions.
+We provide installers for Theia Blueprint to be downloaded (see below) from the Theia webpage.
```

Comment:
I believe "Theia webpage" is unclear. If we are referencing the eclipse-theia page on the eclipse foundation, we can directly link it:

- https://download.eclipse.org/theia/

Reply:
I removed the last part of the sentence.